src/components/image/Image.props.ts

```diff
@@ -0,0 +1,40 @@
+import { TShapeBorderRadius } from '../../foundations/shape';
+
+import TImageSize from './Image.types';
+
+export type PImage = {
+    /**
+     * the src of the image
+     * */
+    source: string;
+    /**
+     * whether the image is full sized or auto
+     * @default auto
+     * */
+    size?: TImageSize;
+    /**
+     * the shape that the image is cropped in
+     * @default 4
+     * */
+    radius?: TShapeBorderRadius;
+    /**
+     * image width
+     * */
+    width?: number | string;
```

Comment:
maybe add a default value her as well? It is `auto`, right?

Reply:
the `size` determines if its auto or full, and you can set a numeric width too, if you don't pass one the width will be 100% driven by the `size` prop